src/css/components/_progress.scss

```diff
@@ -53,6 +54,13 @@
   top: 0;
 }
 
+.video-js .vjs-mouse-display {
```

Comment:
yes. This makes the two displays the same size. I can move it down in the file, if you'd like.